src/tools/rbd/action/Device.cc

```diff
@@ -206,6 +259,14 @@ Shell::Action action_unmap(
   {"device", "unmap"}, {"unmap"}, "Unmap a rbd device.", "",
   &get_unmap_arguments, &execute_unmap);
 
+Shell::Action action_attach(
+  {"device", "attach"}, {"attach"}, "Attach image to nbd device.", "",
```

Comment:
I see, good to know.
I thought its a shortcut to simplify the CLI.
For me:
1.  rbd attach --device /dev/nbd0 rpool/image --options="try-netlink" --device-type nbd
2.  rbd device attach --device /dev/nbd0 rpool/image --options="try-netlink" --device-type nbd

2 looks confusing over 1, because there are more use of `device` strings in the command.

If you still want it to be removed, please let me know.

Thanks!

Reply:
Yes, I think they should be removed.

Reply:
:+1: Sure